.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,74 @@
+Based on the nature of your change, please use the relevant template below and modify to suit your needs.
```

Comment:
Not really, that info is actually very good. My main question is whether it really belongs in a PR template. Maybe it is good to have a reminder about upgrades, but possibly only in the major commit template. (seeming as it really only applies there)